dietpi/dietpi-banner

```diff
@@ -56,6 +56,11 @@
 	#/////////////////////////////////////////////////////////////////////////////////////
 	# Main Loop
 	#/////////////////////////////////////////////////////////////////////////////////////
+	export TERM=${TERM:-dumb}
+	if (( "$TERM" == "dumb" )); then
```

Comment:
exit also gets trigger with dropbear logins.

Heres the value of TERM:
export TERM=${TERM:-dumb}; echo -e "$TERM"
xterm

I tested with the following, works in all cases:
if [ "$TERM" = "dumb" ]; then

Tested with the following to confirm (exit got triggered):
if [ "$TERM" = "xterm" ]; then

If you can test with openssh server, just to verify:
if [ "$TERM" = "dumb" ]; then

Thanks tdaitx.